src/wasm/wasm-emscripten.cpp

```diff
@@ -944,6 +1030,12 @@ AsmConstWalker fixEmAsmConstsAndReturnWalker(Module& wasm) {
   return walker;
 }
 
+void EmscriptenGlueGenerator::fixEmAsm() {
```

Comment:
No one else seems to call `fixEmAsmConstsAndReturnWalker`, so let's merge the two.

`fixEmAsmConstsAndReturnWalker` was always kind of a kludgy thing, I forget the exact reason it was used in more than one place, but that seems to no longer be the case.

Reply:
Done.